packages/plans-grid-next/src/components/actions.tsx

```diff
@@ -147,18 +146,16 @@ const PlanFeatures2023GridActions = ( {
 
 	let actionButton = (
 		<Plans2023Tooltip
-			text={ translate( 'Please contact support to downgrade your plan.' ) }
+			text={ tooltipText ?? translate( 'Please contact support to downgrade your plan.' ) }
```

Comment:
NIT - Is the tooltip still relevant on mobile if we are rendering the same text below the button?

Reply:
It is not relevant. What do you suggest? I'm concerned that adding a check to hide this on mobile might decrease readability.